src/sound_chatot.c

```diff
@@ -0,0 +1,187 @@
+#include "sound_chatot.h"
+#include "sound.h"
+#include "constants/species.h"
+#include "math_util.h"
+#include "unk_02004A44.h"
+#include <nitro/spi/ARM9/mic.h>
```

Comment:
This is already included in <nitro/spi.h>, so is not necessary here. (the way this project is set up, all the nitro sdk and nitrosystem headers are recursively included via command line argument)